R/import_tag.R

```diff
@@ -18,8 +18,15 @@ import_tag <- function(
     tag_id,
     file_url
 ) {
-
+### Comments
+# maybe an optional parameter for a webhook?
+# not sure if this is needed w/ import_tag_status -- but maybe some users like to monitor all their data flows in one place
+​
     auth_reach()
+    ### Comments
+    # is it best practice to authenticate within the function? prevents users from authenticating w/ methods other than Sys.getenv
+    # when managing keys for multiple campaigns, may want to be able to loop through a list of credentials
+    # maybe an optional parameter for access_token, then run auth_reach() if not found?
```

Comment:
I think this makes a lot of sense. Definitely up for adding an `access_token` arguement and setting up some logic to authenticate using the Sys.getenv() method if not provided. 

This has me thinking there could be an improvement made to auth_reach() to make it print something like "You are now authenticated for [campaign]", so that when folks don't supply an access_token they know where they're authenticating for. Need to mull that over some more, but also that's not necessary for this PR.